### ISSUE TITLE
backspace acts as delete in xterm and konsole

### ISSUE THREAD

**Issue by dmt4** [vigna/ne#10]
When "ne" is run in xterm or konsole, pressing the backspace key acts exactly as pressing the delete key, i.e. deletes a character on the right side instead of the left side. In urxvt on the other hand it seems to behave correctly.
I have tried on archlinux and opensuse with the same result.

**Comment by utoddl** [comment]
This is covered extensively in the (now ancient) [http://tldp.org/HOWTO/BackspaceDelete/index.html](http://tldp.org/HOWTO/BackspaceDelete/index.html). It's amazing that after all these years so many distributions still ship bad configurations. Alas, this is not a problem (1) with ne or (2) that ne can fix.

**Comment by dmt4** [comment]
Sorry if I hit a sore point. The strange thing is according to that document my konsole behaves as a standard emulator, i.e. backspace keys emits 127/DEL in the linked test. On any other text handling program (emacs, vim, python interpreter, whatever else...) I have used, backspace behaves as expected, the only exception is 'ne'. Now I am not claiming that the problem is with ne, since you certainly have deeper understanding of the mess that key mapping is. I'm only asking for a suggestion which will make ne behave as expected while not breaking emacs and others. 
Thanks

**Comment by vigna** [comment]
Did you follow the instruction of the HOWTO? Did you check which terminal you are using and whether it interprets 127/DEL as backspace?

**Comment by utoddl** [comment]
Do you have a ~/.ne/.keys file, and if so, what's in it?

**Comment by dmt4** [comment]
I did follow the howto and things looked as they should be, kbs from infocmp konsole matched the emitted codes. Now looked more carefully and turns out TERM is set to xterm but it should have been konsole!

As for ~/.ne/.keys, previously I did try to come up with some workaround with it but eventually deleted it.

Thanks for saying "..whether it interprets 127/DEL as backspace" I had only observed the reverse of this being true. Thanks again!
